velox/connectors/hive/HiveConnectorUtil.cpp

```diff
@@ -931,6 +946,18 @@ void updateParquetWriterOptions(
               *hiveConfig->config(), core::QueryConfig::kSessionTimezone);
   }
 
+  parquetWriterOptions->parquetDataPageVersion =
+      getParquetDataPageVersion(
+          *sessionProperties,
+          parquet::WriterOptions::kParquetSessionDataPageVersion)
+          .has_value()
+      ? getParquetDataPageVersion(
```

Comment:
Please use if-else. Ternary operators are not very easy to read.

Reply:
Fair, rewrote.